### DISCUSSION TITLE
Windowslike TM [3.5 inch] [portrait]

### DISCUSSION THREAD

**Discussion by seythegamer** [mathoudebine/turing-smart-screen-python#584] [Themes]
Hello,
There's my theme that does meet my needs: 
 * simple
 * good looking
 * readable

Networking stats has max value set as '90000000', and '9000000' but you can edit it for ur need (my internet speed in peak is about 80M/s download or 8M/s upload so it still leaves a little gap in the graph so when im using full potential of my speed its not just drawn as big rectangle, but you can see where it ends) - you can also turn on autoscale, set max value as 999999999999999 and it may fit u.

![preview](https://github.com/user-attachments/assets/633f3c28-f527-4bc9-9c16-377908bd6d0b)

Download:
[3.5-windowslike-tm.zip](https://github.com/user-attachments/files/17335195/3.5-windowslike-tm.zip)

**Comment by RenanMMoura** [comment]
nice one! very clean and informative! do you have a landscape version?

**Reply by seythegamer** [reply]
Thanks,
for now ill stick to the portrait mode

but i did quick visualization how it could look like in landscape mode for 3.5" while still providing same readability

![image](https://github.com/user-attachments/assets/a7442d0c-a7ae-4fe1-8d5c-fda0a596dfda)

**Comment by Ntonik7** [comment]
Hello  seythegamer

I just download your theme but i have a little problem with the network stats 

I have ethernet cable and the stats is always 0kb/s , when i connect a usb wifi adapter works fine
Try from config settings all adapters from ethernet and turn on autoscale but none working(in other themes etc Cyberpunk ethernet works fine)

Thanks for your time

**Reply by seythegamer** [reply]
![image](https://github.com/user-attachments/assets/c767998c-4523-4d73-92ac-d0d496755abc)
set it to your prefference

you can swap between ethernet and wifi and viceversa

i just set it up in theme as wifi adapter, and im too lazy to change it to ethernet so i changed it in the configuration :)

**Reply by Ntonik7** [reply]
> ![image](https://private-user-images.githubusercontent.com/77732535/376060501-c767998c-4523-4d73-92ac-d0d496755abc.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjg4NDUzNTUsIm5iZiI6MTcyODg0NTA1NSwicGF0aCI6Ii83NzczMjUzNS8zNzYwNjA1MDEtYzc2Nzk5OGMtNDUyMy00ZDczLTkyYWMtZDBkNDk2NzU1YWJjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMTMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDEzVDE4NDQxNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWU2OTEwNzIwNTc3NmJjMzRhYWRmZjhjNTFkNzY2MWIzNDUwN2YyNGU4NzdlYTNiZDI4NDc2M2E3ZWFlNTg4MGUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.2O8jPOxPZfRWWAoUOmKLH0bcodRGyIhn7pLeOuxNvTc) set it to your prefference
> 
> you can swap between ethernet and wifi and viceversa

Again thanks for your time and quick help

You have right it works fine now

**Comment by emilius902** [comment]
Hi
I'm new and i like your theme but i never add a theme and idk how to do... can you help me plz

**Reply by GNKyrios** [reply]
the files need to be in a folder in res/themes (on the main folder of the proyect)

**Reply by emilius902** [reply]
i dit that and it did'nt worked

**Reply by GNKyrios** [reply]
i just dragged the folder inside the zip ino the location i told you and it shows on the app
![image](https://github.com/user-attachments/assets/2ca06cc4-46a6-48c5-92fa-b5f342010acf)

**Reply by emilius902** [reply]
i think the problem is that i don't have same program
![image](https://github.com/user-attachments/assets/caee3b1e-cb82-495d-8e30-761426ca7338)
i have this

**Reply by GNKyrios** [reply]
oh yes, that is the original software, this one is a custom made with python to replace the original, they use different file types for the themes so they are not compatible.

You have to follow this tutorial to run this proyect: [https://github.com/mathoudebine/turing-smart-screen-python/wiki/System-monitor-%3A-how-to-start](url), download python 3.11 because it didnt work with later versions for me. Also replace this file because i had to modify something to make it run as well.
[configure.zip](https://github.com/user-attachments/files/18101907/configure.zip)
replace the original one with the one inside this .zip

Its kind of complicated to have it run the first time, but if you follow everything on the wiki it should work.
Another tip, when you are on console (cmd) you usually have to start the python comands like this one:
`python3 -m pip install -r requirements.txt` as `py.exe -m pip install -r requirements.txt` and also start it from the folder you unziped the project files (since it reads "requirements.txt" to download all that it needs)

Yeah, i spent like 2 days figuring it out since it didnt download the files neither it ran without finding what was causing an error, but with all that it launched, you have to troubleshoot some things though.

**Comment by Pokomko** [comment]
Hi, can anyone help me with this(i mean the way graphs look)?
![photo_2025-01-25_01-41-05 (2)](https://github.com/user-attachments/assets/13b87566-165d-4fb8-9a9d-7c6c5178b4c0)
Fix network and disk stats disposition i think i can do myself, but i don't know how to fix graphs. I checked built-in theme LandscapeModernDevice35, which also has a graph, and it works fine there.

**Reply by seythegamer** [reply]
hey,

im not sure why, but the new update of turing-smart-screen broke this theme, i tried to paste the graph from another source and edit it to match what i intented them to look like, but still same thing happen

ill try to make it from scratch, and will see what happens then

**Reply by christianled59** [reply]
Hi, I fixed it. Fixes the line sizes and the network and disk text. 
[3.5-windowslike-Fixed.zip](https://github.com/user-attachments/files/19611938/3.5-windowslike-Fixed.zip)

**Reply by seythegamer** [reply]
tysm,
was too lazy to fix it lol

**Reply by paul222008** [reply]
> Hi, I fixed it. Fixes the line sizes and the network and disk text. [3.5-windowslike-Fixed.zip](https://github.com/user-attachments/files/19611938/3.5-windowslike-Fixed.zip)

Thanks for fixing it. Can you add CPU & GPU Wattage?

**Comment by pkpk1230** [comment]
Hello. I really like this task manager style!
I have a request—would it be possible to create a version where the "DISK" section is replaced with an "FPS" section?

**Reply by pkpk1230** [reply]
https://github.com/mathoudebine/turing-smart-screen-python/releases/tag/3.3.1

Sorry to say, but after checking the requirements, it turns out that FPS display is currently limited to AMD GPUs only.
So sad

**Reply by paul222008** [reply]
Love this theme! Thanks for making it

Maybe it's possible to add the CPU &  GPU Wattage? That might be useful

**Comment by approxit1** [comment]
made a landscape version if anyone wanted one
<img width="480" height="320" alt="preview 1" src="https://github.com/user-attachments/assets/4e242354-11f4-4295-8c20-c3447b066af2" />
[tm-landscape.zip](https://github.com/user-attachments/files/21702672/tm-landscape.zip)

**Comment by Yakh0nt** [comment]
Does anyone have something like this for the official program? I'd be incredibly grateful!

**Reply by c0ldJS** [reply]
Most everyone here that is following and using this repo/project have broken away from the official software offerings.

I suppose it's possible that someone out there keeps their box stocked with both flavors -- this project and the software provided by the screen manufacturer -- and flips between them. If so, maybe one of them can come forward. But you'll probably want to mention the software you're targeting (at best) or the model of screen you're using (at minimum). The software that comes with displays depends on who manufactures and sells it.
